pyproject.toml

```
@@ -111,11 +114,14 @@ package-dir = { "" = "src" }
 where = ["src"]
 
 [project.entry-points.'nomad.plugin']
-parser_entry_point = "nomad_json_parser.parsers:parser_entry_point"
-schema_package_entry_point = "nomad_json_parser.schema_packages:schema_package_entry_point"
-
-app_entry_point = "nomad_json_parser.apps:app_entry_point"
+jsonmapper_entry = "nomad_json_parser:jsonmapper_entry"
+jsonmapper_parser = "nomad_json_parser:jsonmapper_parser"
+mappedjson_entry = "nomad_json_parser:mappedjson_entry"
+mappedjson_parser = "nomad_json_parser:mappedjson_parser"
+examplemain_entry = "nomad_json_parser:examplemain_entry"
+examplereference_entry = "nomad_json_parser:examplereference_entry"
```

See comments in `src/nomad_json_parser/__init__.py`
```suggestion
json_mapper_schema_package = "nomad_json_parser.schema_packages:json_mapper_schema_package"
json_mapper_parser = "nomad_json_parser.parsers:json_mapper_parser"
mapped_json_parser = "nomad_json_parser.parsers:mapped_json_parser"
example_schema_package = "nomad_json_parser.schema_packages:example_schema_package"
```